i18n/__init__.py

```diff
@@ -6,7 +6,7 @@
 
 from . import config
 
-__version__ = '1.1.1'
+__version__ = '1.2.0'
```

Comment:
I'm just curious; is this being manually updated or do we have automatic semantic versioning of some sort for this repo?

Reply:
It's suprisingly hard to answer this 😅. Here's what I've checked:

 - No semantic-release integration was found: https://github.com/search?q=repo%3Aopenedx%2Fi18n-tools+semantic&type=code
 - No workflow appears to support it: https://github.com/openedx/i18n-tools/tree/master/.github/workflows
 - It seems to be manually updated: https://github.com/openedx/i18n-tools/commits/master/i18n/__init__.py

I'll file an issue for that.